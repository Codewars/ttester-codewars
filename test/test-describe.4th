\ Copyright 2019 nomennescio
s" group of tests" describe#{
  s" single test" it#{
    <{ 0 -> 0 }>
  }#
}#
