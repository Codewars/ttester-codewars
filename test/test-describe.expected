
<DESCRIBE::>group of tests

<IT::>single test

<PASSED::>Test Passed

<COMPLETEDIN::>0.021 ms

<COMPLETEDIN::>0.038 ms
